warnng `SPVM/blib/lib/SPVM/Builder/include/spvm_implement.h:2683:1: warning: non-void function does not return a value [-Wreturn-type]`

when i make SPVM, I got the following warning.

```
SPVM/blib/lib/SPVM/Builder/include/spvm_implement.h:2683:1: warning: non-void function does not return a value [-Wreturn-type]
}
^
```

the function `static inline int SPVM_IMPLEMENT_CALL_INTERFACE_METHOD(...)` doesn't have return sentence.

i suggest to add return or change function return type `int` to `void`

my env 
```
% uname -mrs
Darwin 19.6.0 x86_64

% gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/c++/4.2.1
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin19.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

--- a/lib/SPVM/Builder/include/spvm_implement.h
+++ b/lib/SPVM/Builder/include/spvm_implement.h
@@ -2667,7 +2667,7 @@ static inline void SPVM_IMPLEMENT_RETURN_MULNUM_DOUBLE(SPVM_ENV* env, SPVM_VALUE
 #define SPVM_IMPLEMENT_CALL_CLASS_METHOD(env, stack, error, method_id, args_stack_length) (error = env->call_method(env, stack, method_id, args_stack_length))
 #define SPVM_IMPLEMENT_CALL_INSTANCE_METHOD(env, stack, error, method_id, args_stack_length) (error = env->call_method(env, stack, method_id, args_stack_length))
 
-static inline int SPVM_IMPLEMENT_CALL_INTERFACE_METHOD(SPVM_ENV* env, SPVM_VALUE* stack, void* object, const char* interface_name, const char* method_name, int32_t args_stack_length, int32_t* error, char* tmp_buffer) {
+static inline void SPVM_IMPLEMENT_CALL_INTERFACE_METHOD(SPVM_ENV* env, SPVM_VALUE* stack, void* object, const char* interface_name, const char* method_name, int32_t args_stack_length, int32_t* error, char* tmp_buffer) {
   int32_t entity_method_id = env->get_instance_method_id(env, stack, object, method_name);
   if (entity_method_id < 0) {
     memset(tmp_buffer, sizeof(tmp_buffer), 0);

